### DISCUSSION TITLE
Gradient calculation failure

### DISCUSSION THREAD

**Discussion by moesphere** [casadi/casadi#3194] [Q&A] [answered]
Hello,

I am trying to build the gradient for a least squares problem with CasADi. I tried to formulate a minimal working example, but did not succeed so far.

The problem is the following:
![image](https://github.com/casadi/casadi/assets/19595387/a15ab090-d860-4e78-9956-da13f7ea6f0b)

Gradient calculation for the squared errors is successful, but fails if I use the sum of squared errors. I have also tried different implementations of the sum, but they all fail (either with the `jacobian` or the `gradient` function). Does someone know  why this fails or should i provide an MWE?

**Comment by jgillis** [comment, marked as answer]
Known issue fixed yesterday on main. 3.6.3 will include a fix in the coming days

**Reply by moesphere** [reply]
Great, thanks for the fast reply!
